docs/.vitepress/vitepress/components/theme-editor/ep-theme-primary.vue

```diff
@@ -0,0 +1,30 @@
+<script lang="ts" setup>
+import { computed, ref } from 'vue'
```

Comment:
```suggestion
import { computed } from 'vue'
```

Reply:
This is a draft and will continue to be modified.
You can review it when it is done.